### ISSUE TITLE
RSS feed lists dummy hourly ghost updates

### ISSUE THREAD

**Issue by stefanb** [WeblateOrg/weblate#5700]
**Describe the issue**

RSS contains a series of hourly dummy updates.

RSS feed: https://hosted.weblate.org/exports/rss/sledilnik/website/it/ lists many hourly updates by user _None_.

The file https://github.com/sledilnik/website/commits/master/src/locales/it.json has no updates in that period, but IS currently the last modified file in the repository (git HEAD, as can be seen on https://github.com/sledilnik/website/commits/master )

Weblate project is configured to publish the changes hourly.

It is a bit annoying to see those updates in our RSS feed (we have all languages feed subscribed in a Slack channel).
Other languages do not have this problem.

Similarly https://hosted.weblate.org/projects/sledilnik/website/#history lists hourly _Committed changes_.

**I already tried**

- [X] I've read and searched [the docs](https://docs.weblate.org/) and did not find the answer there.
      If you didn’t try already, try to search there what you wrote above.
- [X] I've searched [the issues](https://github.com/WeblateOrg/weblate/issues) for _rss_, _feed_ and _xml_, and did not find any related issue there.

**To Reproduce the issue**

1. Go to https://hosted.weblate.org/exports/rss/sledilnik/website/it/ 
2. See repeated items in xml (formatted for clarity):
```xml
<?xml version="1.0" encoding="utf-8"?>
<rss version="2.0"
    xmlns:atom="http://www.w3.org/2005/Atom">
    <channel>
        <title>Recent changes in Sledilnik.org/COVID-19 website — Italian</title>
        <link>https://hosted.weblate.org/projects/sledilnik/website/it/</link>
        <description>All recent changes made using Weblate in Sledilnik.org/COVID-19 website — Italian.</description>
        <atom:link href="https://hosted.weblate.org/exports/rss/sledilnik/website/it/" rel="self"></atom:link>
        <language>en</language>
        <lastBuildDate>Sat, 20 Mar 2021 09:18:16 +0000</lastBuildDate>
        <item>
            <title>Committed changes</title>
            <link>https://hosted.weblate.org/projects/sledilnik/website/it/</link>
            <description>Committed changes at 2021-03-20 09:18:16.430469+00:00 on Sledilnik.org/COVID-19 website — Italian by &lt;span title="None"&gt;None&lt;/span&gt;</description>
            <dc:creator xmlns:dc="http://purl.org/dc/elements/1.1/">&lt;span title="None"&gt;None&lt;/span&gt;</dc:creator>
            <pubDate>Sat, 20 Mar 2021 09:18:16 +0000</pubDate>
            <guid>https://hosted.weblate.org/projects/sledilnik/website/it/</guid>
        </item>
        <item>
            <title>Committed changes</title>
            <link>https://hosted.weblate.org/projects/sledilnik/website/it/</link>
            <description>Committed changes at 2021-03-20 08:18:21.352749+00:00 on Sledilnik.org/COVID-19 website — Italian by &lt;span title="None"&gt;None&lt;/span&gt;</description>
            <dc:creator xmlns:dc="http://purl.org/dc/elements/1.1/">&lt;span title="None"&gt;None&lt;/span&gt;</dc:creator>
            <pubDate>Sat, 20 Mar 2021 08:18:21 +0000</pubDate>
            <guid>https://hosted.weblate.org/projects/sledilnik/website/it/</guid>
        </item>
        <item>
            <title>Committed changes</title>
            <link>https://hosted.weblate.org/projects/sledilnik/website/it/</link>
            <description>Committed changes at 2021-03-20 07:18:33.577444+00:00 on Sledilnik.org/COVID-19 website — Italian by &lt;span title="None"&gt;None&lt;/span&gt;</description>
            <dc:creator xmlns:dc="http://purl.org/dc/elements/1.1/">&lt;span title="None"&gt;None&lt;/span&gt;</dc:creator>
            <pubDate>Sat, 20 Mar 2021 07:18:33 +0000</pubDate>
            <guid>https://hosted.weblate.org/projects/sledilnik/website/it/</guid>
        </item>
        <item>
            <title>Committed changes</title>
            <link>https://hosted.weblate.org/projects/sledilnik/website/it/</link>
            <description>Committed changes at 2021-03-20 06:18:14.996400+00:00 on Sledilnik.org/COVID-19 website — Italian by &lt;span title="None"&gt;None&lt;/span&gt;</description>
            <dc:creator xmlns:dc="http://purl.org/dc/elements/1.1/">&lt;span title="None"&gt;None&lt;/span&gt;</dc:creator>
            <pubDate>Sat, 20 Mar 2021 06:18:14 +0000</pubDate>
            <guid>https://hosted.weblate.org/projects/sledilnik/website/it/</guid>
        </item>
        <item>
            <title>Committed changes</title>
            <link>https://hosted.weblate.org/projects/sledilnik/website/it/</link>
            <description>Committed changes at 2021-03-20 05:18:17.070381+00:00 on Sledilnik.org/COVID-19 website — Italian by &lt;span title="None"&gt;None&lt;/span&gt;</description>
            <dc:creator xmlns:dc="http://purl.org/dc/elements/1.1/">&lt;span title="None"&gt;None&lt;/span&gt;</dc:creator>
            <pubDate>Sat, 20 Mar 2021 05:18:17 +0000</pubDate>
            <guid>https://hosted.weblate.org/projects/sledilnik/website/it/</guid>
        </item>
        <item>
            <title>Committed changes</title>
            <link>https://hosted.weblate.org/projects/sledilnik/website/it/</link>
            <description>Committed changes at 2021-03-20 04:18:16.462217+00:00 on Sledilnik.org/COVID-19 website — Italian by &lt;span title="None"&gt;None&lt;/span&gt;</description>
            <dc:creator xmlns:dc="http://purl.org/dc/elements/1.1/">&lt;span title="None"&gt;None&lt;/span&gt;</dc:creator>
            <pubDate>Sat, 20 Mar 2021 04:18:16 +0000</pubDate>
            <guid>https://hosted.weblate.org/projects/sledilnik/website/it/</guid>
        </item>
        <item>
            <title>Committed changes</title>
            <link>https://hosted.weblate.org/projects/sledilnik/website/it/</link>
            <description>Committed changes at 2021-03-20 03:18:17.621278+00:00 on Sledilnik.org/COVID-19 website — Italian by &lt;span title="None"&gt;None&lt;/span&gt;</description>
            <dc:creator xmlns:dc="http://purl.org/dc/elements/1.1/">&lt;span title="None"&gt;None&lt;/span&gt;</dc:creator>
            <pubDate>Sat, 20 Mar 2021 03:18:17 +0000</pubDate>
            <guid>https://hosted.weblate.org/projects/sledilnik/website/it/</guid>
        </item>
        <item>
            <title>Committed changes</title>
            <link>https://hosted.weblate.org/projects/sledilnik/website/it/</link>
            <description>Committed changes at 2021-03-20 02:18:36.553647+00:00 on Sledilnik.org/COVID-19 website — Italian by &lt;span title="None"&gt;None&lt;/span&gt;</description>
            <dc:creator xmlns:dc="http://purl.org/dc/elements/1.1/">&lt;span title="None"&gt;None&lt;/span&gt;</dc:creator>
            <pubDate>Sat, 20 Mar 2021 02:18:36 +0000</pubDate>
            <guid>https://hosted.weblate.org/projects/sledilnik/website/it/</guid>
        </item>
        <item>
            <title>Committed changes</title>
            <link>https://hosted.weblate.org/projects/sledilnik/website/it/</link>
            <description>Committed changes at 2021-03-20 01:18:15.224760+00:00 on Sledilnik.org/COVID-19 website — Italian by &lt;span title="None"&gt;None&lt;/span&gt;</description>
            <dc:creator xmlns:dc="http://purl.org/dc/elements/1.1/">&lt;span title="None"&gt;None&lt;/span&gt;</dc:creator>
            <pubDate>Sat, 20 Mar 2021 01:18:15 +0000</pubDate>
            <guid>https://hosted.weblate.org/projects/sledilnik/website/it/</guid>
        </item>
        <item>
            <title>Committed changes</title>
            <link>https://hosted.weblate.org/projects/sledilnik/website/it/</link>
            <description>Committed changes at 2021-03-20 00:18:12.624453+00:00 on Sledilnik.org/COVID-19 website — Italian by &lt;span title="None"&gt;None&lt;/span&gt;</description>
            <dc:creator xmlns:dc="http://purl.org/dc/elements/1.1/">&lt;span title="None"&gt;None&lt;/span&gt;</dc:creator>
            <pubDate>Sat, 20 Mar 2021 00:18:12 +0000</pubDate>
            <guid>https://hosted.weblate.org/projects/sledilnik/website/it/</guid>
        </item>
    </channel>
</rss>
```

**Expected behavior**

No such dummy repeated items.

**Screenshots**

in Slack:
![image](https://user-images.githubusercontent.com/319826/111865663-9ab3a280-8968-11eb-8651-190002f83fd6.png)

git log:
![image](https://user-images.githubusercontent.com/319826/111865806-6e4c5600-8969-11eb-84d8-33edfb085ef8.png)

**Exception traceback**

**Server configuration and status**

Hosted Weblate version 4.5.2-dev

**Additional context**

**Comment by nijel** [comment]
There was a bug in the commit code, it is fixed in fcd1ebec6f6838615f1bc3a3cb3f6a8ef9e44b4a.

**Comment by stefanb** [comment]
Hooray! 🎉 
This seems to have fixed the problem, thanks!
